core/src/main/java/org/mobilitydata/gtfsvalidator/notice/Notice.java

```diff
@@ -16,13 +16,15 @@
 
 package org.mobilitydata.gtfsvalidator.notice;
 
+import com.google.common.base.CaseFormat;
 import com.google.common.base.Joiner;
 import java.util.Collections;
 import java.util.Map;
 import java.util.Objects;
 
 /** Base class for all notices produced by GTFS validator. */
 public abstract class Notice {
+  private static String NOTICE_SUBSTRING = "notice";
```

Comment:
This variable should also be `final`. I would also rename it to `NOTICE_SUFFIX`.